### DISCUSSION TITLE
Malformed F1AP packets on wireshark using CU-DU split

### DISCUSSION THREAD

**Discussion by kimdh98** [srsran/srsRAN_Project#855] [General Help] [answered]
Hi, 
I have set up srsRAN using the CU-DU split with open5gs configuration.
I am trying to analyze the F1AP packets during the attach procedure of a COTS UE.
The UE seems to attach successfully and can access the internet through the 5G connection, but when I open the .pcap file on wireshark, all the packets show up as malformed.
Furthermore, although I cannot verify its contents, but it seems on Wireshark that UE setup procedures have failed, which is strange since the connection seems stable on UE.

The SDR and UE in use are USRP B210 (without GPSDO) and Google Pixel 6.
I attached a screenshot of the Wireshark capture for reference.
Any insights would be greatly appreciated!

![Screenshot from 2024-10-10 18-29-46](https://github.com/user-attachments/assets/8ddea4d4-256f-45e6-9bcc-675453c2337c)

**Comment by pgawlowicz** [comment, marked as answer]
@kimdh98 could you check with the newest project version?

**Reply by kimdh98** [reply]
I updated to latest version, and it seems to display it correctly now. Thanks!
